File: Software University/09) Java Advanced - January 2019/14. Defining Classes - Exercises/04. Raw Data/CarParts/Cargo.java
package CarParts;

public class Cargo {

    private Integer weight;

    private String type;

    public Cargo(Integer weight, String type) {
        setWeight(weight);
        setType(type);
    }

    public Integer getWeight() {
        return weight;
    }

    private void setWeight(Integer weight) {
        this.weight = weight;
    }

    public String getType() {
        return type;
    }

    private void setType(String type) {
        this.type = type;
    }
}

File: Software_University/04_PHP_Web_Development_Basics/10) OOP Basics - Exercises/06. Raw Data/Cargo.php
<?php

namespace Van\Classes;

class Cargo
{
    /**
     * @var int
     */
    private $weight;

    /**
     * @var string
     */
    private $type;

    /**
     * Cargo constructor.
     *
     * @param int $weight
     * @param string $type
     */
    public function __construct(int $weight, string $type)
    {
        $this->weight = $weight;
        $this->type = $type;
    }

    /**
     * @return int
     */
    public function getWeight(): int
    {
        return $this->weight;
    }

    /**
     * @param int $weight
     */
    public function setWeight(int $weight)
    {
        $this->weight = $weight;
    }

    /**
     * @return string
     */
    public function getType(): string
    {
        return $this->type;
    }

    /**
     * @param string $type
     */
    public function setType(string $type)
    {
        $this->type = $type;
    }
}